src/components/Table/GridTableApi.ts

```diff
@@ -96,19 +104,42 @@ export class GridTableApiImpl<R extends Kinded> implements GridTableApi<R> {
   }
 
   public getSelectedRowIds(kind?: string): string[] {
-    return this.getSelectedRows(kind).map((row: any) => row.id);
+    return this.getSelectedRowIdsImpl(kind ?? undefined);
+  }
+
+  // impl with required param for computedFn
+  private getSelectedRowIdsImpl(kind: string | undefined): string[] {
```

Comment:
The `computedFn` implementation requires a constant number of parameters on every invocation, which, dunno, not sure why, seems kinda annoying imo, but to keep our current API of "kind is optional", we have to materialized the `kind = undefined` before calling into the computed fn.